web/src/components/PoiListItem.tsx

```diff
@@ -38,6 +38,11 @@ const Distance = styled.div`
   font-size: clamp(0.55rem, 1.6vh, ${props => props.theme.fonts.hintFontSize});
 `
 
+const Category = styled.div`
+  font-size: clamp(0.55rem, 1.6vh, ${props => props.theme.fonts.hintFontSize});
```

Comment:
Why such an complicated way to define the font-size? Can't this be simplified?
Also as this same computation with magic number is duplicated, please move it to one location and import it in the other.

Reply:
because its the way how to use different paddings and font-size using the viewport width (CSS3 feature). If we use equal paddings and font-size it will look odd in different desktop screen sizes. 
I will define the sizes and reuse them